app/models/work.rb

```diff
@@ -0,0 +1,44 @@
+WORK_CATEGORIES = ["book", "album", "movie"]
+
+class Work < ApplicationRecord
+  has_many :votes, dependent: :destroy
+  has_many :users, through: :votes
+
+  validates :title, presence: true, uniqueness: true
+
+  def self.by_category(category_string)
+    return Work.where(category: category_string)
+  end
+
+  def self.spotlight
+    works = self.all
+
+    most_votes = works.max_by do |work|
+      work.votes.count
+    end
+
+    return most_votes
+  end
```

Comment:
This kind of thing works, but it's more efficient to use the database to sort and filter data.  Consider using `.where`, `.order` and `.limit` to do this kind of thing.

You can see a [blog post about it here](https://tomkadwill.com/2017/12/18/counting-or-sorting-by-an-association-in-a-rails-sql-query.html).